data/lads.csv

```diff
@@ -344,5 +344,5 @@ E07000100,St Albans,,2011-12-01,terminated
 E07000101,Stevenage,,2011-12-01,terminated
 E07000104,Welwyn Hatfield,,2011-12-01,terminated
 E08000020,Gateshead,,2011-12-01,terminated
-z,Fenland and East Cambridgeshire combined,,,
+E07000010 / E07000009,Fenland / East Cambridgeshire combined,,,
```

Comment:
Should this still have 'combined' in? Does the GSS guidance say either way?

Reply:
Good spot, meant to leave that off. Have removed it now. Here's the relevant guidance that covers this: 

> Any statistical publication using combined geographical areas should define these areas clearly, in terms of their names and geography codes, for example Cornwall/Isles of Scilly, E06000052/E06000053.